ifrs17/Utils/EqualityComparers.ipynb

```diff
@@ -46,16 +72,11 @@
                 "\n",
                 "\n    public bool Equals(RawVariable x, RawVariable y) =>",
                 "\n        x.AccidentYear == y.AccidentYear && x.AmountType == y.AmountType && x.DataNode == y.DataNode && x.AocType == y.AocType && ",
-                "\n        x.Novelty == y.Novelty && x.EstimateType == y.EstimateType && (IgnoreValues ? true : x.Values.SequenceEqual(y.Values)); // TODO: enable Precision",
+                "\n        x.Novelty == y.Novelty && x.EstimateType == y.EstimateType && (IgnoreValues ? true : x.Values.SequenceEqual(y.Values, Precision));",
                 "\n",
-                "\n    public int GetHashCode(RawVariable v) => IgnoreValues  ",
-                "\n                                            ? HashCode.Combine(v.AccidentYear, v.AmountType, v.DataNode, v.AocType, v.Novelty) ",
-                "\n                                            : HashCode.Combine(v.AccidentYear, v.AmountType, v.DataNode, v.AocType, v.Novelty, v.Values);",
+                "\n    public int GetHashCode(RawVariable v) => 0;",
```

Comment:
Is this correct?

Reply:
yes, being the equality comparer based on a identity property matching function, the hash code is not used here. Any proper method to compute hash would only sacrifice performance without added value.